### PR TITLE
Vup.Vup version 0.13.3

### PR DESCRIPTION
**Pull request by lukehmcc**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101558)

### CODE REVIEW

**Comment by lukehmcc** [comment]
@microsoft-github-policy-service agree

File: manifests/v/Vup/Vup/0.13.3/Vup.Vup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Vup.Vup
+PackageVersion: 0.13.3
+Installers:
+- InstallerUrl: https://github.com/vup-app/vup/releases/download/v0.13.3/VupSetup-0.13.3-BETA-Windows.exe
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: 32D3A6D1AF03B67406E0CB4A91962479ADADD9F6A91B42A6318562A49A367B4A
```
Comment (stephengillie) [inline manifests/v/Vup/Vup/0.13.3/Vup.Vup.installer.yaml:10]:
```suggestion
  InstallerSha256: 32D3A6D1AF03B67406E0CB4A91962479ADADD9F6A91B42A6318562A49A367B4A
  InstallerSwitches:
    Custom: /AllUsers
```

Comment (lukehmcc) [reply manifests/v/Vup/Vup/0.13.3/Vup.Vup.installer.yaml:10]: Will this fix the installer failing without user input? The only two things that stop it on my local machine are windows defender and choosing a user. I presume the /AllUsers fixes the second one.

**Comment by stephengillie** [comment]
I'm not able to find the cause for this error. It installs and runs normally on a Windows 11 VM.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation failed with the seemingly-truncated:
> Standard error:

**Pull request merged**